IFC4x3/Properties/c/CurrentCarryingCapacity_3C4YXmdtnBXOYpmGiU46y8/Documentation.md

```diff
@@ -0,0 +1 @@
+Maximum value of electric current which can be carried continuously by a conductor, a device or an apparatus, under specified conditions without its steady-state temperature exceeding a specified value [IEV 826.11.13]
```

Comment:
If reuse, I would propose modifying the description of `CurrentCarryingCapacity`.